CONTRIBUTING.md

```diff
@@ -50,7 +42,7 @@ We keep a running list of changes to this application in the [CHANGELOG](CHANGEL
   - [React Router](https://github.com/ReactTraining/react-router/tree/v3/docs)
 
 ### Workflow
-- Open branches off main repo due to secure CircleCI env vars.
+- Open branches off main repo (as opposed to a fork) due to secure CircleCI env vars.
```

Comment:
Thanks for the suggestion. I forget what the issue with the environment variables is so I just updated the text to reflect what you said and removed that point.